src/tools/ScenarioMeasurement/Startup/WindowsTraceSession.cs

```diff
@@ -102,9 +102,9 @@ private void InitWindowsKeywordMaps()
             clrKeywords[TraceSessionManager.ClrKeyword.Startup] = ClrPrivateTraceEventParser.Keywords.Startup;
         }
 
-        public void EnableUserProvider(string provider)
+        public void EnableUserProvider(string provider, TraceEventLevel verboseLevel)
```

Comment:
should this have a default value for `verboseLevel`?

Reply:
sure!

Reply:
sure!